pkg/version/version.go

```diff
@@ -0,0 +1,7 @@
+package version
+
+// Version is the current version number, e.g: v0.0-251-ge29921f
+var Version string
+
+// Revision is the git revision, e.g: e29921fb682f610b1663efaee278e8d5551b9051
+var Revision string
```

Comment:
Revision is quite a broad term and doesn't seem to be used anywhere. Do we need it? maybe GitHash?

Reply:
I was going to clear up some other areas where we burn the revision into the image etc, but never did. Happy to rename that!